Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -242,7 +247,8 @@ public void PostProc_ScannerEngine_Failure()
               ]
             }
             """
-                .ToEnvironmentLineEndings());
+                .ToEnvironmentLineEndings(),
+            Arg.Any<CmdLineArgPropertyProvider>());
```

Comment:
Same here: assert the content of the passed CmdLineArgPropertyProvider with Arg.Is